cli/src/commands/extensions/api.rs

```diff
@@ -420,7 +420,8 @@ async fn parse_lockfile(
     let lockfile_type = match lockfile_type {
         Some(lockfile_type) => lockfile_type,
         None => {
-            let (packages, package_type) = parse::get_packages_from_lockfile(Path::new(&lockfile))?;
+            let (packages, package_type) =
+                parse::get_packages_from_lockfile(Path::new(&lockfile), false)?;
```

Comment:
I chose to err on the side of caution here because extensions should probably retain control over what gets printed to stdout. But maybe we should expose this as an option in the extension API?